src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -25,8 +25,8 @@
 {
     protected $name;
     protected $parent;
-    protected $normalizationClosures = array();
-    protected $finalValidationClosures = array();
+    protected $normalizationCallbacks = array();
+    protected $finalValidationCallbacks = array();
```

Comment:
Works without, but of course it makes sense to keep the old property declarations as well. Restored now, with deprecation notes.